app/src/main/java/uk/co/ribot/androidboilerplate/ui/base/BaseActivity.java

```diff
@@ -23,7 +22,7 @@
 
     private static final String KEY_ACTIVITY_ID = "KEY_ACTIVITY_ID";
     private static final AtomicLong NEXT_ID = new AtomicLong(0);
-    private static final Map<Long, ConfigPersistentComponent> sComponentsMap = new HashMap<>();
+    private static final LongSparseArray<ConfigPersistentComponent> sComponentsMap = new LongSparseArray<>();
```

Comment:
This line is too long, our guidelines state that the length of a line should not exceed 100 characters. https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md#2215-line-length-limit

Reply:
Okey, check again.